### ISSUE TITLE
Can this be updated?

### ISSUE THREAD

**Issue by travibe** [FarisHijazi/Rarbg-Enhancer-UserScript#30]
I was so upset when rarbg was destroyed. But, as of today... its back up and running. Or at least its proxies. Using your script really spoiled me badly, and I would beg of you to update it so it can work on the new sites... PLEASE?

these are the new proxies they have listed on their site (https://rarbg.tw/)

https://www.rarbggo.to/
[https://rarbgproxy.to](https://www.rarbgproxy.to/)
[https://rarbgmirror.org](https://www.rarbgproxy.to/)
[https://rarbgaccessed.org](https://www.rarbgproxy.to/)
[https://rarbgget.org](https://www.rarbgproxy.to/)

Any help would be greatly, deeply, passionately appreciated. Thank you!

**Comment by FarisHijazi** [comment]
Currently it works with rargb.to
Tell any specific issues you face and I'll do my best.

**Comment by FarisHijazi** [comment]
are you having any specific issues with those mirrors? @travibe ?

it works great on: https://rargb.to/

**Comment by FarisHijazi** [comment]
added support for https://www.rarbggo.to/

**Comment by travibe** [comment]
Hey buddy,

Im sorry for taking so long to respond to your messages. I have been unable
to get emails for some time but i must say how happy i was to see you
responding back to me about your userscript. Youve been very kind all this
time from helping make that customized script to begin with, and now taking
the time to update it.
I am about to be at my desktop in a few minutes and i am going to install
it and take it for a spin. Im sure its working now that youve looked at it.
If not, ill tell you of any complications.
Thank you so much for taking your time and helping me out! You are a good
man!

**Comment by travibe** [comment]
Hey FarisHijazi,

I tried your script last night and I can't get the damn thing to work to
save my life. I have attached an image to this email showing what it looks
like on my end. I have disabled my UblockO extension as well to make sure
it wasn't that. I have placed your script as #1 on Tampermonkey script
placement so it supersedes all other scripts, and disabled other scipts for
RARBG. I dont know if its working on its mirror site
https://www2.rarbggo.to/? Can you provide any pointers? Thank you so much.

**Comment by FarisHijazi** [comment]
can you try it on rargb.to and if it doesn't work, please open the devtools (F12) and go to the console and send me the errors as a screenshot.

maybe the issue is that tampermonkey scripts work? maybe it's the new tampermonkey manifest v3 update?

**Comment by FarisHijazi** [comment]
do any tampermonkey scripts work? maybe it's the new tampermonkey manifest v3 update?

**Comment by travibe** [comment]
yes i have other scripts that run on other sites that do successfully work
like Pagetual etc. heres what i have installed for your reference. I
dont seem to have issue with any other scripts that i know of. And your
script is the only one configured to run on RARBG.

**Comment by FarisHijazi** [comment]
ok try it on rargb.to and screenshot the errors in the console, and tell me your exact tampermonkey version and browser version

**Comment by travibe** [comment]
I saw there was quite a few of them so I copied and pasted.... hope this
works, if not let me know.


Uncaught TypeError: Cannot read properties of null (reading 'style')
    at document.onmousemove (header.js:2:27)
149b635a6de17f1e483fc1571db55dc5.js:1


       Failed to load resource: net::ERR_SSL_PROTOCOL_ERROR
429713581ee618a8c732b1beae3c7094.js:1


       Failed to load resource: net::ERR_SSL_PROTOCOL_ERROR
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:97609
Script: RARBG Enhancer meta: Object
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:99395
search engine: google Object
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98584
Binding keys to categories:
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98586
"v": "Movies"
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98586
"s": "TV shows"
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98586
"m": "Music"
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98586
"w": "Software"
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98586
"x": "XXX"
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98586
"g": "Games"
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98586
"n": "Non XXX"
header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:3
jQuery.Deferred exception: checkbox is not defined ReferenceError: checkbox
is not defined
    at HTMLDocument.main
(chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98221:21)
    at e
(chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:3:29453)
    at Proxy.t
(chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:3:29755)
    at Rt (<anonymous>:9:89)
    at t.<computed> (<anonymous>:66:214)
    at Rt (<anonymous>:9:89)
    at r (<anonymous>:24:484)
    at <anonymous>:25:112
    at <anonymous>:76:476
    at m (<anonymous>:76:495) undefined
k.Deferred.exceptionHook @
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:3
userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:3
Uncaught ReferenceError: checkbox is not defined
    at HTMLDocument.main
(userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:98221:21)
    at e
(userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:3:29453)
    at Proxy.t
(userscript.html?name=RARBG-Enhancer.user.js&id=e30d44e3-b48d-404f-8152-9b0f0875114a:3:29755)
    at Rt (<anonymous>:9:89)
    at t.<computed> (<anonymous>:66:214)
    at Rt (<anonymous>:9:89)
    at r (<anonymous>:24:484)
    at <anonymous>:25:112
    at <anonymous>:76:476
    at m (<anonymous>:76:495)
21header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
d.agkn.com/pixel/6644/?che=1705689011156&sk=214610604726002577316:1


       Failed to load resource: net::ERR_NAME_NOT_RESOLVED
www2.rarbggo.to/:1 Refused to execute script from '
https://track2.securedvisit.com/sync/1540_03681?id=IBM2jSZHo4bkZtYrQri-7pxW'
because its MIME type ('image/gif') is not executable.
100header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
header.js:2 Uncaught TypeError: Cannot set properties of null (setting
'innerHTML')
    at overlib (header.js:2:100)
    at HTMLAnchorElement.onmouseover ((index):122:253)
3header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
28header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
document.onmousemove @ header.js:2
header.js:6 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at nd (header.js:6:19)
    at HTMLAnchorElement.onmouseout ((index):122:253)
nd @ header.js:6
onmouseout @ (index):122
header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
document.onmousemove @ header.js:2
header.js:2 Uncaught TypeError: Cannot set properties of null (setting
'innerHTML')
    at overlib (header.js:2:100)
    at HTMLAnchorElement.onmouseover ((index):126:247)
overlib @ header.js:2
onmouseover @ (index):126
9header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
document.onmousemove @ header.js:2
header.js:6 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at nd (header.js:6:19)
    at HTMLAnchorElement.onmouseout ((index):126:247)
nd @ header.js:6
onmouseout @ (index):126
header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
document.onmousemove @ header.js:2
header.js:2 Uncaught TypeError: Cannot set properties of null (setting
'innerHTML')
    at overlib (header.js:2:100)
    at HTMLAnchorElement.onmouseover ((index):130:331)
overlib @ header.js:2
onmouseover @ (index):130
11header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
document.onmousemove @ header.js:2
header.js:6 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at nd (header.js:6:19)
    at HTMLAnchorElement.onmouseout ((index):130:331)
nd @ header.js:6
onmouseout @ (index):130
header.js:2 Uncaught TypeError: Cannot set properties of null (setting
'innerHTML')
    at overlib (header.js:2:100)
    at HTMLAnchorElement.onmouseover ((index):134:262)
overlib @ header.js:2
onmouseover @ (index):134
5header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
document.onmousemove @ header.js:2
header.js:6 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at nd (header.js:6:19)
    at HTMLAnchorElement.onmouseout ((index):134:262)
nd @ header.js:6
onmouseout @ (index):134
header.js:2 Uncaught TypeError: Cannot set properties of null (setting
'innerHTML')
    at overlib (header.js:2:100)
    at HTMLAnchorElement.onmouseover ((index):138:307)
overlib @ header.js:2
onmouseover @ (index):138
header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
document.onmousemove @ header.js:2
header.js:6 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at nd (header.js:6:19)
    at HTMLAnchorElement.onmouseout ((index):138:307)
nd @ header.js:6
onmouseout @ (index):138
47header.js:2 Uncaught TypeError: Cannot read properties of null (reading
'style')
    at document.onmousemove (header.js:2:27)
document.onmousemove @ header.js:2
beacon:9


       GET https://match.prod.bidr.io/cookie-sync/ro?rndcb=3818233459 400
(Bad Request)
ro:1


       Failed to load resource: the server responded with a status of 400
(Bad Request)
ro:1


       Failed to load resource: the server responded with a status of 400
(Bad Request)
ro:1


       Failed to load resource: the server responded with a status of 400
(Bad Request)
ro:1


       Failed to load resource: the server responded with a status of 400
(Bad Request)
p.adsymptotic.com/d/px/?_pid=16257&_psign=5a9f251662be469b9732c38b03f11952&_redirect=https%3A%2F%2Fpippio.com%2Fapi%2Fsync%3Fpid%3D710202%26it%3D1%26iv%3D%24%7BUUID%7D&_rand=05059256:1


       GET
https://p.adsymptotic.com/d/px/?_pid=16257&_psign=5a9f251662be469b9732c38b03f11952&_redirect=https%3A%2F%2Fpippio.com%2Fapi%2Fsync%3Fpid%3D710202%26it%3D1%26iv%3D%24%7BUUID%7D&_rand=05059256
net::ERR_NAME_NOT_RESOLVED

**Comment by travibe** [comment]
Hey Bud,

Somehow, last night, I figured out the secret to getting the script to work
for me. The very first setting "Thumbnail Link" I had sent to "Magnet Link"
but got desperate and decided to just try all different variables of the
settings. Turns out if I switched this setting to "img", the script appears
correctly, functions properly, and everyone is happy! That is seriously the
only thing that was wrong. I don't understand why, but don't really care,
as long as I can use this script! Love it! Thank you so much for your
continued assistance with this for so long. I really appreciate you!
